src/main/java/com/stripe/model/Order.java

```diff
@@ -82,6 +85,10 @@ public class Order extends ApiResource implements HasId, MetadataStore<Order> {
   @SerializedName("created")
   Long created;
 
+  /** The credits applied to the Order. At most 10 credits can be applied to an Order. */
+  @SerializedName("credits")
+  List<Order.Credit> credits;
```

Comment:
Should that be just `Credit`? Not sure if there is any difference.

Reply:
Not sure why it generated the fully qualified version, but `Order.Credit` is fine here! Good question.